server/database.js

```diff
@@ -8,43 +8,36 @@ const users = require('./json/users.json');
  * @param {String} email The email of the user.
  * @return {Promise<{}>} A promise to the user.
  */
-const getUserWithEmail = function(email) {
-  let user;
+const getUserWithEmail = function (email) {
   for (const userId in users) {
-    user = users[userId];
+    const user = users[userId];
```

Comment:
Hey, @andydlindsay.  Could this be refactored to:

```suggestion
    const user = users[userId];
    const resolvedUser = null;
    if (user?.email.toLowerCase() === email?.toLowerCase()) {
      resolvedUser = user;
    }
    return resolvedUser;
};
```
```